compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/ProgramFileConstants.java

```diff
@@ -25,9 +25,9 @@ public class ProgramFileConstants {
 
     public static final int MAGIC_NUMBER = 0xBA1DA4CE;
     public static final short VERSION_NUMBER = 50;
-    public static final int BIR_VERSION_NUMBER = 68;
-    public static final short MIN_SUPPORTED_VERSION = 68;
-    public static final short MAX_SUPPORTED_VERSION = 68;
+    public static final int BIR_VERSION_NUMBER = 69;
```

Comment:
We only need to update the BIR version, if the actual BIR was changed. In other words, reading/writing. Here, with your change, you only have used an already available value to populate another reference. Hence, the actual BIR didn't change

Reply:
Got it.